test/support/updating_permissions_helpers.rb

```diff
@@ -35,6 +35,7 @@ def assert_update_permissions(application, grantee, grant: [], revoke: [])
 
     click_button "Update permissions"
 
+    assert_flash_content("Permissions updated")
     assert_flash_content(grant.map(&:name))
```

Comment:
These could be combined (`assert_flash_content(["Permissions updated", *grant.map(&:name)])`) so that the flash element is only grabbed once, but the clarity of this is probably more important!

Reply:
As we've discussed over the past couple of weeks, I'm definitely one for more readability in tests!